src/Upgrade/Application/Strategy/ReleaseApp/Validator/Threshold/PathThresholdValidator.php

```diff
@@ -38,7 +38,8 @@ public function __construct(ConfigurationProvider $configurationProvider)
     public function validate(ReleaseGroupDtoCollection $releaseGroupDtoCollection): void
     {
         $softThreshold = $this->configurationProvider->getSoftThresholdBugfix();
-        if ($releaseGroupDtoCollection->getCommonModuleCollection()->getPathAmount() > $softThreshold) {
+        $pathAmount = $releaseGroupDtoCollection->getCommonModuleCollection()->getPathAmount();
```

Comment:
typo here. Should  be `patch` and `getPatchAnount`

Reply:
Thanks fixed